src/components/CardList/card-list.jsx

```diff
@@ -175,14 +175,14 @@ export class CardList extends Component {
     const indicators = {
       info: 0,
       warning: 1,
-      'hard-stop': 2,
+      'critical': 2,
```

Comment:
Can you remove the single quotes around `'critical'`? They were needed when this was `hard-stop` (due to the dash), but now we can remove them which puts it inline with the other indicator values.